connect-base/2.4/Dockerfile

```diff
@@ -42,6 +44,14 @@ RUN mkdir "$KAFKA_CONNECT_PLUGINS_DIR" "$EXTERNAL_LIBS_DIR"
 #
 RUN docker-maven-download apicurio "$APICURIO_VERSION" ead18a95038adca54e91b7f253717eb7
 RUN docker-maven-download central org/jolokia jolokia-jvm "$JOLOKIA_VERSION" d489d62d1143e6a2e85a869a4b824a67
+RUN docker-maven-download central io/opentelemetry opentelemetry-api "$OPENTELEMETRY_VERSION" e198a9568ce31a82faaa26f328388e89
```

Comment:
it would be good to download it into external lib directory or newly created dir (e.g. `otel`) and enable it only on demand, i.e. introduce new env. variable [here](https://github.com/debezium/container-images/blob/main/connect-base/2.4/docker-entrypoint.sh#L41) and later on eventually, if enabled, create the symlink for the jar file by calling `set_connector_additonal_resource_availability` (see [here](https://github.com/debezium/container-images/blob/main/connect-base/2.4/docker-entrypoint.sh#L142))

Reply:
@vjuranek 
I made a modification, but I don't see how the downloaded file can then be found in the otel subdirectory. I can't test on the PC I'm currently using, I'll check that later.

Reply:
@vidourem the bit will be donwloaded into a separated directory. If nothing happens, they won't be on the class path. If the user enables OTEL via env. variable, calling `set_connector_additonal_resource_availability` with this env. variable enabled will result into a creating a symbolic link from connector directory to OTEL libs and therefore they will be included on the class path.